vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -121,67 +118,58 @@ spec:
+1. From the AppBuilder go to `Configuration -> File Browser` 
+2. Migrate to the `public` folder and create a new folder with the same name as the bundle (the value is in the `code` field available in the `/bundle/descriptor.yaml` file of the blueprint project) or use a custom name such as `demo-widget`.
 
-> **Warning**
->
-> If you choose to use a custom folder, remember to update the references in the customUI of the widget later
+3. Upload all the resources in the `/bundle/resources` folder of the project into the new folder
 
-2. Upload all the resources available in the `/bundle/resources` folder
-of the blueprint project into the folder you created above in the App Builder file browser.
-> **Warning**
->
-> You can decide to recreate the folder structure to be the same as the one in `/bundle/resources` or not, but you need to update the
-> references in the customUI / configUI of the widget later if you choose a different folder structure.
+::: tip Warning
+You can duplicate the folder name and structure of `/bundle/resources`, but if you do not, you need to update the references to them in the customUI/configUI of the widget later.
+:::
 
-3. Create the widget. In the App Builder go to `Components → Micro frontends & Widgets`.
+4. In the App Builder go to `Components → Micro frontends & Widgets`  
 
-4. Select `Add`
+5. Select `Add` to create the widget
 
-5. Set whatever title you want
+6. Enter the following with your choices for `anyname`:
+- `Title: anyname1` → enter both English and Italian languages
+- `Code: anyname2` → dashes are not allowed
+- `Group: Free Access`
+- `Icon`:  → upload an icon of your choice
 
-6. For the customUI copy the one created in one of the widgets you generated from the blueprint.
-As an example, the customUI for the `detailWidget` of the conference entity is available in
+7. Under customUI, copy from one of the widgets you generated with the Blueprint.
+For example, the customUI for the `detailWidget` of the Conference entity is available in
 `/bundle/ui/widgets/conference/detailsWidget/conference-details-widget.ftl`
 
-> **Warning**
->
-> Remember to update all the references in the customUI to use the custom folder structure you defined
-
-7. Update the service url to match the location where you deployed the ingress for your microservice if you used a path different than the default. For example,
-
-```
-Update the service-url value with the relative path of your service if you're deploying a bundle
-		Ex. <car-table service-url="/newBp/api"  />
-	Or use the full path if you're deploying the BE and FE individually
-		Ex. <car-table service-url="http://newbp-plugin-entando.192.168.64.7.nip.io/newBp/api"/>
-```
+8. Update the service url to match the location where you deployed the ingress for your Microservice if you used a path different than the default. 
```

Comment:
URL ? 
microservice lowercase?
would reorder: "If your path differs from the default, upgrade the service URL to match the location where the ingress for your microservice was deployed."